### PR TITLE
Fix staging

### PR DESCRIPTION
**Pull request by zsinx6**
#### What
Fix CORS and createsuperuser script.

#### Why
To be able to configure CORS using env in terraform.

### CODE REVIEW

File: jobbergateapi2/config.py [hunk 1]
```diff
@@ -18,7 +18,7 @@ class Settings(BaseSettings):
     SERVERLESS_STAGE: str = Field("staging")
     SERVERLESS_REGION: str = Field("eu-north-1")
     S3_BASE_PATH: str = Field("jobbergate-resources")
-    BACKEND_CORS_ORIGINS: List[AnyHttpUrl] = []
+    BACKEND_CORS_ORIGINS: str = Field("[]")
```
Comment (matheushent) [inline jobbergateapi2/config.py:21]: what is the reason for changing it?

Comment (zsinx6) [reply jobbergateapi2/config.py:19]: Take a look here: https://github.com/omnivector-solutions/infrastructure/pull/18

Comment (matheushent) [reply jobbergateapi2/config.py:19]: can you add an example in anywhere? e.g. `"['https://example1.com', 'https://example2.com']"`. It seems dummy but can save some understanding in the future

**Review by jamesbeedy — APPROVED** [review]
Nice. +1

**Pull request merged**